libs/books/feature/src/lib/book-search/book-grid/book-grid-item/book-grid-item.component.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { SharedTestingModule } from '@tmo/shared/testing';
+import { BooksFeatureModule } from '../../../books-feature.module';
+import { BookGridItemComponent } from './book-grid-item.component';
+
+describe('BookGridItemComponent', () => {
+  let component: BookGridItemComponent;
+  let fixture: ComponentFixture<BookGridItemComponent>;
+  const book = {
+      id: `${Date.now()}b`,
+      title: `Some Title ${Math.floor(Math.random() * 9999)}b`,
+      authors: ['a', 'b', 'c'],
+      publisher: 'Some Publisher',
+      publishedDate: new Date().toDateString(),
+      description: 'n/a',
+      isAdded: false
+    };
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [BooksFeatureModule, SharedTestingModule]
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(BookGridItemComponent);
+    component = fixture.componentInstance;
+    component.book = book;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('shall render book as expected', () => {
+    const componentBook = fixture.debugElement.componentInstance.book;
```

Comment:
You know what this is already! It's defined as `book` in the scope of the `describe`.